### PR TITLE
Coding Practice 25

### PR DESCRIPTION
**Pull request by waghmaresuyog**

### CODE REVIEW

File: simple/src/codingPractice/AdditionOfMatrix.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package codingPractice;
+
+public class AdditionOfMatrix {
```
Comment (RajashreeSonawane) [inline simple/src/codingPractice/AdditionOfMatrix.java:3]: space between method and class

File: simple/src/codingPractice/AdditionOfMatrix.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            for (int columnIndex = 0; columnIndex < 3; columnIndex++) {
+                //Addiction fo Two Matrix
+                sumOfMatrix[rowIndex][columnIndex] = matrixFirst[rowIndex][columnIndex] + matrixSecond[rowIndex][columnIndex];
+                System.out.print(sumOfMatrix[rowIndex][columnIndex] + " ");
+            }
+            System.out.println();
+        }
+
+    }
+
```
Comment (RajashreeSonawane) [inline simple/src/codingPractice/AdditionOfMatrix.java:39]: same here

File: simple/src/codingPractice/AdditionOfMatrix.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        add(matrixFirst, matrixSecond);
+    }
+
+    public static void add(int matrixFirst[][], int matrixSecond[][]) {
+
+        int sumOfMatrix[][] = new int[3][3];
+        for (int rowIndex = 0; rowIndex < 3; rowIndex++) {
+            for (int columnIndex = 0; columnIndex < 3; columnIndex++) {
+                //Addiction fo Two Matrix
+                sumOfMatrix[rowIndex][columnIndex] = matrixFirst[rowIndex][columnIndex] + matrixSecond[rowIndex][columnIndex];
```
Comment (RajashreeSonawane) [inline simple/src/codingPractice/AdditionOfMatrix.java:32]: Please code formatting

File: simple/src/codingPractice/AdditionOfMatrix.java (hunk 1 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/AdditionOfMatrix.java:3]: done

File: simple/src/codingPractice/AdditionOfMatrix.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                System.out.print(sumOfMatrix[rowIndex][columnIndex] + " ");
+            }
+            System.out.println();
+        }
+
```
[the remark this answers is not in the document]
Comment (waghmaresuyog) [reply simple/src/codingPractice/AdditionOfMatrix.java]: done

File: simple/src/codingPractice/AdditionOfMatrix.java (hunk 2 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/AdditionOfMatrix.java]: done

File: simple/src/codingPractice/AdditionOfMatrix.java (hunk 3 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/AdditionOfMatrix.java]: done

**Review by RajashreeSonawane — APPROVED** [review]

**Review by prashantkawa — APPROVED** [review]

**Pull request merged**